src/EventExport/Format/TabularData/TabularDataEventFormatter.php

```diff
@@ -683,6 +687,26 @@ function ($tariffs, $cardSystemPrices) {
                 },
                 'property' => 'bookingAvailability',
             ],
+            'videos.url' => [
+                'name' => 'videos URL',
+                'include' => function ($event) {
+                    if (!property_exists($event, 'videos') || !is_array($event->videos)) {
```

Comment:
You could pass `$event` to `formatVideo` instead of `array $videos`.
That would make it possible to do the check `if (!property_exists($event, 'videos') || !is_array($event->videos))` inside `formatVideo`

Reply:
Fixed